src/Workspaces/Core/Portable/Serialization/ISupportTemporaryStorage.cs

```diff
@@ -13,6 +13,6 @@ namespace Microsoft.CodeAnalysis.Serialization
     /// </summary>
     internal interface ISupportTemporaryStorage
     {
-        IEnumerable<ITemporaryStreamStorageInternal>? GetStorages();
+        IReadOnlyList<ITemporaryStreamStorageInternal>? GetStorages();
```

Comment:
💭 A bit surprised this isn't `ImmutableArray<T>`

Reply:
yeah, i'll change to that.

Reply:
ah, it's stored as a value in a CWT.  it was a bit too annoying to change, so i'm going to keep as is for now.